source/postprocess/gravity_point_values.cc

```diff
@@ -603,12 +632,15 @@ namespace aspect
         prm.enter_subsection ("Gravity calculation");
         {
           if ( (prm.get ("Sampling scheme") == "uniform distribution") || (prm.get ("Sampling scheme") == "map") )
-            sampling_scheme = uniform_distribution;
+            sampling_scheme = map;
           else if ( (prm.get ("Sampling scheme") == "list of points") || (prm.get ("Sampling scheme") == "list") )
             sampling_scheme = list_of_points;
+          else if (prm.get ("Sampling scheme") == "fibonacci spiral")
+            sampling_scheme = fibonacci_spiral;
           else
             AssertThrow (false, ExcMessage ("Not a valid sampling scheme."));
-          quadrature_degree_increase = prm.get_integer ("Quadrature degree increase");
+          quadrature_degree_increase = prm.get_double ("Quadrature degree increase");
```

Comment:
Keep this line in its old state, "Quadrature degree increase" is an integer, not a double.